sktime/base/_base.py

```diff
@@ -177,6 +182,12 @@ def save(self, path=None):
                 path="/home/stored/estimator" then a zip file `estimator.zip` will be
                 stored in `/home/stored/`.
 
+        serialization_format: str, default = "pickle"
+            Module to use for serialization.
+            The available options are present under
```

Comment:
I would either spell it out, or print the `SERIALIZATION_FORMATS` constant in the docstring.
There are only two anyway.

Reply:
Using a variable in the docstring would make things trickier with `__doc__()`. I might as well spell them here again, thanks.